### ISSUE TITLE
Allow to save current browser (don't ask for 2FA code again) in period of time

### ISSUE THREAD

**Issue by nick-hoang** [Offroadcode/Umbraco-2FA#11]

**Comment by PeteDuncanson** [comment]
Yep, this certainly something we want. Again should be opt in as believe it or not some clients don't want it. To me it should be in there but getting the initial stuff working was hard enough. Now we've had time to settle this is certainly something that should be in there.

I'm pondering just getting this into V8 so its out the box.

**Comment by PeteDuncanson** [comment]
Raised this for the record https://github.com/umbraco/Umbraco-CMS/issues/3361
